octoprint_homeassistant/__init__.py

```diff
@@ -936,6 +1025,12 @@ def on_event(self, event, payload):
                 allow_queueing=True,
             )
 
+        if (
+            event == Events.PLUGIN_PSUCONTROL_PSU_STATE_CHANGED
```

Comment:
Will `Events.PLUGIN_PSUCONTROL_PSU_STATE_CHANGED` always be defined, or does this generate a traceback when the plugin isn't installed and active?

Reply:
At the moment, this line generates a traceback for users not using this plugin.